tests/e2e/managed_services_test.go

```diff
@@ -163,4 +164,60 @@ var _ = Describe("Managed services tests", Label(tests.LabelSmoke, tests.LabelBa
 			}, testTimeouts[utils.ManagedServices]).Should(Succeed())
 		})
 	})
+
+	It("should properly handle replace update strategy", func(ctx SpecContext) {
```

Comment:
Perhaps we could add another E2E that runs in cloud environments and tries to create and update a load balancer.

@mnencia

Reply:
It's a good idea, but the test could be very flaky (load balancers are very slow to provision) and may require specific per-CSP tests to be meaningful.

@fcanovai, what do you think?